Libraries/Azure.psm1

```diff
@@ -2616,7 +2616,9 @@ Function Add-DefaultTagsToResourceGroup {
 		}
 		Add-ResourceGroupTag -ResourceGroup $ResourceGroup -TagName BuildURL -TagValue $BuildURLTag
 		# Add test name.
-		Add-ResourceGroupTag -ResourceGroup $ResourceGroup -TagName TestName -TagValue $currentTestData.testName
+		if ($currentTestData.testName) {
```

Comment:
when it is empty?

Reply:
at the beginning of preparing environment and create resource group for tip.